zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -5201,6 +5201,48 @@ paths:
           $ref: "#/components/responses/InternalServerError"
         "503":
           $ref: "#/components/responses/ServiceUnavailable"
+  /element-instances/{elementInstanceKey}/incidents/search:
+    post:
+      tags:
+        - Element instance
+      operationId: searchElementInstanceIncidents
+      summary: Search for incidents of a specific element instance
+      description: |
+        Search for incidents caused by the specified element instance. Supported only for document-based backends (Elasticsearch/OpenSearch).
```

Comment:
❓ Should we describe that this includes incidents of child instances created from this element instance as well?